fast_td3/fast_td3/actors/gnn/egnn2.py

```
@@ -0,0 +1,242 @@
+        :param attention: Whether using attention or not
+        :param normalize: Normalizes the coordinates messages such that:
+                    instead of: x^{l+1}_i = x^{l}_i + Σ(x_i - x_j)phi_x(m_ij)
+                    we get:     x^{l+1}_i = x^{l}_i + Σ(x_i - x_j)phi_x(m_ij)/||x_i - x_j||
+                    We noticed it may help in the stability or generalization in some future works.
+                    We didn't use it in our paper.
+        :param tanh: Sets a tanh activation function at the output of phi_x(m_ij). I.e. it bounds the output of
+                        phi_x(m_ij) which definitely improves in stability but it may decrease in accuracy.
+                        We didn't use it in our paper.
+        """
+
+        super(EGNN, self).__init__()
```

Pylint (4.0.2)

[refactor] Consider using Python 3 style super() without arguments

(R1725)